lib/components/viewers/RouteRow.js

```diff
@@ -69,11 +60,11 @@ export const ModeIconElement = styled.span`
 
 const RouteDetailsContainer = styled.div`
   align-items: center;
-  display: grid;
-  grid-template-columns: repeat(4, auto);
+  display: flex;
   gap: 12px;
-  justify-items: center;
   margin-left: 5px;
+  // Make sure route details always leaves enough room for pattern button
+  max-width: 87%;
```

Comment:
How about, instead, setting a `padding-right` on `RouteRowButton` with value the `PatternButton` width plus the margins around it?

Reply:
Great idea, thank you! Changed in [6384960](https://github.com/opentripplanner/otp-react-redux/pull/969/commits/638496002b716f8d199ae115bd56a122f175a677)